apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/UserAvatar.tsx

```diff
@@ -13,7 +13,7 @@ interface UserAvatarProps {
 
 export const UserAvatar = ({ walletName, isPopup }: UserAvatarProps): React.ReactElement => {
   const [handle] = useGetHandles();
-  const handleImage = handle?.nftMetadata?.image;
+  const handleImage = handle?.profilePic || handle?.nftMetadata?.image;
```

Comment:
```suggestion
  const handleImage = handle?.profilePic;
```
We only want to show the profile pic if it's available, as the larger image doesn't scale well